functions/Get-DbaPlanCache.ps1

```diff
@@ -72,14 +74,10 @@
                 }
 
                 $results = $server.Query($sql)
+                $size = [dbasize]($results.MB*1024*1024)
```

Comment:
Why not just return the raw value of bytes instead of pulling it as MB (which converted it from bytes) and then converting it back?

Reply:
Also I would remove the decimal cast in the query.

Reply:
easier in the refactor ;)

Reply:
Don't think I follow... You mean that type expects MB as input to allow it to refactor to the other units?

Reply:
yah the clear plancache relies on it so i just kept it and hid it